paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AddressElementActivity.kt

```diff
@@ -92,13 +97,22 @@ internal class AddressElementActivity : ComponentActivity() {
                                     InputAddressScreen(viewModel.injector)
                                 }
                                 composable(AddressElementScreen.Autocomplete.route) {
-                                    AutoCompleteScreen(viewModel.injector)
+                                    val autocompleteViewModel: AutocompleteViewModel =
```

Comment:
@skyler-stripe I changed the parameters of the composable, let me know what you think